fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -49,6 +52,8 @@ import org.mozilla.fenix.tabstray.syncedtabs.SyncedTabsListItem
 import org.mozilla.fenix.theme.FirefoxTheme
 import mozilla.components.browser.storage.sync.Tab as SyncTab
 
+private const val PAGER_DELAY = 100L
```

Comment:
In my testing, I'm not seeing that this delay is required to fix the bug. Is that different than what you've been seeing? What motivated it originally? IMO if we can get this in without the delay it will feel a lot less hacky.

Reply:
WAT

So, I tried a NUMBER of different things, and in the beginning of debugging potential solutions, I tried running a `LaunchedEffect` with the forced re-scroll to the correct page, but that didn't seem to do anything. Fast forward to last night, it worked. Out of nowhere. I _thought_ I had tested without the delay, but I'll double check and report back.

Reply:
OK, so taking away the delay made it broken for _every_ rotation to landscape 🤦‍♀️ 

As a footnote, I've never reproduced this in portrait (either when rotating back to portrait, or opening the tabs tray in landscape and then rotating).

Reply:
hrmm, a delay of 50 instead of 100 seems to also work